.mage/dev.go

```diff
@@ -35,6 +36,38 @@ func (Dev) Certificates() error {
 	return execGo("run", path.Join(runtime.GOROOT(), "src", "crypto", "tls", "generate_cert.go"), "-ca", "-host", "localhost,*.localhost")
 }
 
+// Misspell fixes common spelling mistakes in files.
+func (Dev) Misspell() error {
+	if mg.Verbose() {
+		fmt.Printf("Fixing common spelling mistakes in files\n")
+	}
+	return execGo("run", "github.com/client9/misspell/cmd/misspell", "-w", "-i", "mosquitto",
```

Comment:
I don't really like how we're now explicitly listing the files and folders that need to be spellchecked. Can we perhaps do this with globbing?

Reply:
There's no obvious/simple way to do this via globbing and it would require explicitly specifying things anyway.
I tried to do it via globbing first, but then decided it's not worth the effort, especially since the files/directories we need to spellcheck are not going to change any time soon.

Reply:
Ok. I'll approve because it's just dev tooling, but we shouldn't start accepting this kind of nastiness in "real" code.

Reply:
I don't see any nastiness here - we need to have a clear distinction between things we can spellcheck and things we can't. For example, we should not all of a sudden start spellchecking `LICENSE` or `go.sum`, but e.g. `.editorconfig` or `Makefile` might have comments to spellcheck. We can build a bunch of unnecessary complex globbing solutions here, but there's not point in doing that as we have pretty much constant set of directories/files to spellcheck.